src/hub/vendor/customer.py

```diff
@@ -36,6 +39,50 @@ def run(self) -> None:
         self.send_to_routes(routes, json.dumps(data))
 
 
+class StripeCustomerDeleted(AbstractStripeHubEvent):
+    def run(self) -> None:
+        logger.info("customer deleted", payload=self.payload)
+        customer_id = self.payload.data.object.id
+        uid = self.payload.data.object.metadata.userid
+        deleted_user = g.subhub_deleted_users.get_user(uid)
+        result = {}
+        for d in deleted_user.subscription_info:
+            for k in d.keys():
+                result[k] = result.get(k, 0) + d[k]
```

Comment:
what does this gem of logic do?

Reply:
This gets the total amount for all subscriptions and sends to the routes so they can tell the customer how much they were paying in the emails.